README.rst

```diff
@@ -29,3 +29,37 @@ Example synapse config:
         #bind_dn:
         #bind_password:
         #filter: "(objectClass=posixAccount)"
+
+Troubleshooting and Debugging
+-----------------------------
+
+`matrix-synapse-ldap3` logging is included in the Synapse homeserver log
+(typically `homeserver.log`). The LDAP plugin log level can be increased to
+`DEBUG` for troubleshooting and debugging by making the following modifications
+to your Synapse server's logging configuration (in `[your.domain].log.config`:
```

Comment:
As the logging configuration can be set through `--log-config` and the usage doesn't show a default saying it is `[your.domain].log.config` is presuming to know the local environment, which it can't.

```
logging:
  -v, --verbose         The verbosity level.
  -f LOG_FILE, --log-file LOG_FILE
                        File to log to.
  --log-config LOG_CONFIG
                        Python logging config file
```